storybookjs/storybook
# Add nunito sans font

Resolve: https://github.com/storybookjs/storybook/issues/20828
Resolve: https://github.com/storybookjs/storybook/issues/16278

## What I did

- I add a few font files
- I add these files to be hosted / copied by storybook via a preset config
- I add references to these font files in the manager builder's template

## How to test

- start a storybook in dev-mode
- open the browser
- expect the network tab to show you the browser loading 2 font files in the manager, followed by a css file.
- expect the font defined in that css files to be used in the manager

repeat the above steps but for a build storybook. (you can use a chromatic deployed storybook for this)
